flink-connectors/flink-connector-pulsar/src/test/java/org/apache/flink/connector/pulsar/source/reader/split/PulsarPartitionSplitReaderTestBase.java

```diff
@@ -198,7 +213,7 @@ void pollMessageAfterTimeout(PulsarPartitionSplitReaderBase<String> splitReader)
         String topicName = randomAlphabetic(10);
 
         // Add a split
-        seekStartPositionAndHandleSplit(splitReader, topicName, 0);
+        handleSplit(splitReader, topicName, 0, MessageId.latest);
```

Comment:
Is this change necessary?

If so, I can see there're two similar sentences remaining (L250 & L303) and perhaps they should be changed also.

Reply:
`seekStartPositionAndHandleSplit` is used for existed topic. `handleSplit` is used for non-existed topic. The real testing logic is finished by @imaffe. I don't want to change it until we find it's necessary.

Reply:
@imaffe Can you confirm this review issue?

Reply:
@syhily so you mean that these two changes are all about non-existed topic, and you leave those for existing topic as is?

Reply:
Here we should use seekStartPositionAndHandleSplit(). The difference is whether we want to do a seek before we call handleSplitChanges. Here we want to receive 0 message in the next poll() call, so we need to seek to the `latest` cursor so no data will be consumed.

Reply:
@tisonkun Yep.